cypress/integration/tests/controls/tags/tag/create.test.ts

```diff
@@ -44,19 +46,20 @@ describe("Tag validations", { tags: "@tier2" }, () => {
         preservecookies();
 
         // Interceptors
-        cy.intercept("POST", "/api/controls/tag*").as("postTag");
-        cy.intercept("GET", "/api/controls/tag*").as("getTag");
+        cy.intercept("POST", "/hub/tag*").as("postTag");
+        cy.intercept("GET", "/hub/tag*").as("getTag");
     });
 
     it("Tag field validations", function () {
         // Navigate to Tags tab and click "Create tag" button
+        selectUserPerspective("Developer");
```

Comment:
I think there is no need in duplicating this line for each test unless you will need to change perspective to Admin. This can be put just to `before` section